sunpy/net/scraper.py

```diff
@@ -182,25 +181,10 @@ def _URL_followsPattern(self, url):
         """
         Check whether the url provided follows the pattern.
         """
-        matches = re.match(self.pattern, url)
-        if matches:
-            return matches.end() == matches.endpos
-        return False
-
-    def _extractDateURL(self, url):
-        """
-        Extracts the date from a particular url following the pattern.
-        """
-        # remove the user and passwd from files if there:
-        url = url.replace("anonymous:data@sunpy.org@", "")
-        datestr, formatstr = "", ""
-        dateinfo = parse(self.pattern, url)
-        for datetimetype in TIME_CONVERSIONS:
-            if datetimetype in self.pattern:
-                formatstr = formatstr + TIME_CONVERSIONS[datetimetype]
-                datestr = datestr + str(dateinfo[datetimetype[1:-4]])
-
-        return Time.strptime(datestr, formatstr)
+        if parse(self.pattern, url):
+            return True
+        else:
+            return False
```

Comment:
This could be simplified to:
```suggestion
        return parse(self.pattern, url)
```

Reply:
i knew i was missing a really cleaner way to do this, thanks!